packages/stylelint-plugin/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@kaizen/stylelint-plugin",
+  "version": "1.0.0",
+  "types": "dist/**/*.d.ts",
+  "main": "dist/src/index.js",
+  "license": "MIT",
+  "files": [
+    "src",
+    "dist",
+    "types"
+  ],
+  "bin": {
+    "kaizen-token-codemod": "./dist/src/cli.js"
```

Comment:
Do you think we'll ever want to distinguish stylelint codemods from eslint (or similar) codemods? Do you think this name is still best in a world where other types of codemods exist?

(I think it's probably fine, but wanted to ask the question and see if you think of anything)

Reply:
In case you see this, I ended up un-exposing the CLI (keeping the program just in case we change our mind).
It was originally intended to be used because of some quirks of stylelint, which we're resolved (LESS and SCSS variables are represented differently in the AST when run with stylelint).

I think having both the CLI and stylelint plugin is unnecessary, for now at least.